src/executables/master.c

```diff
@@ -673,7 +704,24 @@ css_send_to_existing_server (CSS_CONN_ENTRY * conn, unsigned short rid, CSS_SERV
   int name_length, buffer;
   name_length = 1024;
   std::string server_name;
-  SERVER_TYPE type;
+  SERVER_TYPE type = SERVER_TYPE_TRANSACTION;
+  bool conn_established_flag = false;
+
+  // *INDENT-OFF*
+  scope_exit <std::function<void (void)>> log_on_exit ([&conn_established_flag, &server_name, &type]()
+  {
+    if (conn_established_flag == true)
+     {
+        MASTER_ER_LOG_DEBUG (ARG_FILE_LINE, "Client successfully connected to %s server of type %s.", server_name.c_str (),
```

Comment:
Not only clients connect to a server; other servers do too. Better to change the message to "Successfully connected to ..." to avoid confusion.